df.art.xml

```diff
@@ -1082,48 +1082,57 @@
         <enum-item name='MERCENARY'/>
```

Comment:
add a comment to the `enum-type` reading `bay12: Occupation`

Reply:
done